src/node.rs

```diff
@@ -72,28 +74,10 @@ impl From<TransformInternal> for Transform {
 /// General information about scene `Node`.
 #[derive(Clone, Debug, PartialEq)]
 pub struct Node {
-    /// Relative to parent transform.
-    pub transform: Transform,
-    /// World transform (relative to the world's origin).
-    pub world_transform: Transform,
     /// Is `Node` visible by cameras or not?
     pub visible: bool,
-    /// The same as `visible`, used internally.
-    pub world_visible: bool,
+    /// Relative to parent transform.
+    pub transform: Transform,
     /// Material in case this `Node` has it.
```

Comment:
Was this ever useful?

Reply:
wait, why is `Material` here? O_O

Reply:
Why not?!

Reply:
Ah, this is user-exposed node, not the internal one.
Yeah, let it be. The whole reflection thing (with `sync`) I'd consider non-idiomatic, so we shouldn't be too eager to optimize it.